src/modules/fancyzones/editor/FancyZonesEditor/EditorOverlay.xaml.cs

```diff
@@ -91,29 +89,29 @@ public void ShowLayoutPicker()
 
             Content = _layoutPreview;
 
-            _mainWindow.Owner = this;
-            _mainWindow.ShowActivated = true;
-            _mainWindow.Topmost = true;
-            _mainWindow.Show();
+            MainWindow.Owner = null;

```

Comment:
Is the change from "this" to "null" intentional? Can't qualify this is as generic refactoring, so wanted to double check :P

Reply:
It is! I did this so we can move the editor overlay from one screen to the other without affecting the `MainWindow` (which was the case with it having an owner) ~